src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -110,4 +112,30 @@
     </xsd:complexType>
     <xsd:element name="ListCertificatesResponseMessage" type="tns:ListCertificatesResponseMessageType"/>
 
+    <xsd:complexType name="CreateCertificateRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="alias" type="xsd:string"/>
+            <xsd:element name="referenceNumber" type="xsd:string"/>
```

Comment:
Are these the only fields we need?

Reply:
these are the required fields for the DN